cardshifter-server/src/main/java/com/cardshifter/server/model/MainServer.java

```diff
@@ -72,9 +85,9 @@ public Server start() {
         mods.loadExternal(mods.getDefaultModLocation());
         mods.loadExternal(Paths.get(config.getModsDirectory()));
 
-        if(mods.getAvailableMods().isEmpty()) {
+        if (mods.getAvailableMods().isEmpty()) {
         	logger.warn("No mods were loaded from the different folders. Please check your installation.");
-        	throw new IllegalStateException("No mod loaded");
+        	throw new NoModsLoadedException("No mod loaded");
```

Comment:
Well it's really nitpicky since it may be redundant but it doesn't hurt anybody. I'll fix it anyway tonight.